callouts/python/extproc/example/jwt_auth/service_callout_example.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2024 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import jwt
+from jwt.exceptions import InvalidTokenError
+
+from grpc import ServicerContext
+from envoy.service.ext_proc.v3 import external_processor_pb2 as service_pb2
+from extproc.service import callout_server
+from extproc.service import callout_tools
+
+def check_request_header(request_headers, header_key):
+  """Check header of the request."""
+  return next((header.key
+                   for header in request_headers.headers.headers
+                   if header.key == header_key), None)
+
+def extract_jwt_token(context):
+  metadata = dict(context.invocation_metadata())
```

Comment:
isnt the JWT token part of some HTTP header like "Authorization: Bearer <...>" ?